File: T5. C# CONDITIONAL STATEMENTS AND LOOPS (Java)/P11_OddNumber.java
import java.util.Scanner;

public class P11_OddNumber {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        for (int i = 0; i < 10; i++) {
            int inputNumber = Integer.parseInt(in.nextLine());
            inputNumber = Math.abs(inputNumber);
            if (inputNumber % 2 == 0) {
                System.out.println("Please write an odd number.");
            } else {
                System.out.printf("The number is: %d", inputNumber);
                break;
            }
        }
    }
}

File: T5. C# CONDITIONAL STATEMENTS AND LOOPS/P11. OddNumber.cs
using System;

namespace P11._OddNumber
{
    class Program
    {
        static void Main(string[] args)
        {
            for (int i = 0; i < 10; i++)
            {
                int inputNumber = int.Parse(Console.ReadLine());
                inputNumber = Math.Abs(inputNumber);

                if (inputNumber % 2 == 0)
                {
                    Console.WriteLine("Please write an odd number.");
                }
                else
                {
                    Console.WriteLine($"The number is: {inputNumber}");
                    break;
                }
            }
        }
    }
}
